src/Sylius/Bundle/ResourceBundle/Routing/RouteFactoryInterface.php

```diff
@@ -32,11 +32,20 @@ public function createRouteCollection();
      * @param array $requirements An array of requirements for parameters (regexes)
      * @param array $options An array of options
      * @param string $host The host pattern to match
-     * @param string|array $schemes A required URI scheme or an array of restricted schemes
-     * @param string|array $methods A required HTTP method or an array of restricted methods
+     * @param array $schemes A required URI scheme or an array of restricted schemes
```

Comment:
I would change the comment to `A required array of restricted schemes` or remove it at all.